src/main/java/mate/project/store/dto/user/UserRegistrationRequestDto.java

```diff
@@ -14,14 +15,16 @@
 public record UserRegistrationRequestDto(
         @NotNull(message = "Email can't be null")
         @NotBlank(message = "Email can't be empty")
```

Comment:
According to documentation of @NotBlank
"The annotated element must not be null and must contain at least one non-whitespace character. Accepts CharSequence."
So there is no sense in @NotNull when you have @NotBlank